src/frontend/e2e/tests/upload.spec.ts

```diff
@@ -11,19 +11,23 @@ test.describe("Upload sample tests", () => {
   });
 
   fileExtensions.forEach((extenstion) => {
-    test(`Should upload ${extenstion.toUpperCase()} sample file`, async ({
+    test.skip(`Should upload ${extenstion.toUpperCase()} sample file`, async ({
       page,
     }) => {
       const uploadData = {
         dataFile: extenstion,
         samples: UploadSample.getSampleData(),
       };
       await UploadSample.uploadSequencingFiles(page, uploadData);
-      await expect(page.locator("//button[not(contains(@class,'Mui-disabled')) and text()='Continue']")).toBeVisible()
+      await expect(
+        page.locator(
+          "//button[not(contains(@class,'Mui-disabled')) and text()='Continue']"
+        )
+      ).toBeVisible();
     });
   });
 
-  test(`Should validate collection dates`, async ({ page }) => {
+  test.skip(`Should validate collection dates`, async ({ page }) => {
```

Comment:
why are we skipping these?

Reply:
@phoenixAja there is a defect here https://app.shortcut.com/genepi/story/216597/filtering-sample-by-collection-date-last-3-months-brings-older-record

Reply:
oh got it, would it be possible to add a todo/ comment explaining?